src/devtoolsPanel.ts

```diff
@@ -365,6 +365,36 @@ export class DevToolsPanel {
             `;
     }
 
+    private getCdnHtmlForWebview() {
+        let cdnBaseUrl = this.config.devtoolsBaseUri;
+        if (!cdnBaseUrl) {
+            // Not provided, calculate based on config
+            // TODO: CDP call to determine actual hash and stuff
+            cdnBaseUrl = this.config.useLocalEdgeWatch ? 'http://localhost:3000/vscode_app.html' : 'https://devtools.invalid';
+        }
+        const hostPath = vscode.Uri.file(path.join(this.extensionPath, 'out', 'host_beta', 'host.bundle.js'));
+        const hostUri = this.panel.webview.asWebviewUri(hostPath);
+
+        const stylesPath = vscode.Uri.file(path.join(this.extensionPath, 'out', 'common', 'styles.css'));
+        const stylesUri = this.panel.webview.asWebviewUri(stylesPath);
+
+        // the added fields for "Content-Security-Policy" allow resource loading for other file types
+        return `
+            <!doctype html>
+            <html>
+            <head>
+                <meta http-equiv="content-type" content="text/html; charset=utf-8">
+                <meta name="referrer" content="no-referrer">
+                <link href="${stylesUri}" rel="stylesheet"/>
+                <script src="${hostUri}"></script>
+            </head>
```

Comment:
We'll want to figure out what the right CSP is to put back here. Ideally we're only allowing the minimum needed capabilities to load and communicate with the frame.